src/com/lftechnology/java/training/srijan/basics/Calculator.java

```diff
@@ -0,0 +1,150 @@
+package com.lftechnology.java.training.srijan.basics;
+
+import java.util.logging.Logger;
+
+/**
+ * Write a program that calculates percentage and GPA of 5 students. User will
+ * enter the marks of each 5 students from the keyboard. Each of the students
+ * has following attributes: name, class, rollnum, and marks obtained in 5
+ * subjects. Calculate the GPA and show the student name and GPA by
+ * position.[higher GPA should display on top]
+ * 
+ * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+ *
+ */
+
+public class Calculator {
+	private float subject1;
+	private float subject2;
+	private float subject3;
+	private float subject4;
+	private float subject5;
+	private static final Logger LOGGER = Logger.getLogger(Calculator.class.getName());
+
+	private String[] nameOfStudent = new String[5];
+	private float[] gpaOfStudent = new float[5];
+	private int countStudent = 0;
+	private int countGpa = 0;
+
+	public Calculator(float subject1, float subject2, float subject3, float subject4, float subject5) {
+		this.subject1 = subject1;
+		this.subject2 = subject2;
+		this.subject3 = subject3;
+		this.subject4 = subject4;
+		this.subject5 = subject5;
+	}
+
+	public void setSubjectMarks(float subject1, float subject2, float subject3, float subject4, float subject5) {
```

Comment:
you haven't used this method. Also set method should be create for each variable